.github/workflows/docker.yml

```diff
@@ -175,6 +183,7 @@ jobs:
       platform: aarch64
       runsOn: ARM64
     needs: [aarch64-scipy]
+    if: github.repository_owner == 'jupyter' && ${{ !contains(github.event.head_commit.message, '[FAST_BUILD]') }}
```

Comment:
This line was completely missing (it's a bug).
It worked, because if some job is skipped (because of if:), and another job needs it, then this job will be skipped as well.

It means essentially we can only specify if: in some of the jobs, but explicit is better than implicit.

Reply:
Solved in main